go-controller/pkg/cluster/gateway_localnet.go

```diff
@@ -170,14 +194,26 @@ func initLocalnetGateway(nodeName string,
 		ovn.OvnNodeGatewayVlanID:     fmt.Sprintf("%d", config.Gateway.VLANID),
 		ovn.OvnNodeGatewayIfaceID:    ifaceID,
 		ovn.OvnNodeGatewayMacAddress: macAddress,
-		ovn.OvnNodeGatewayIP:         localnetGatewayIP,
-		ovn.OvnNodeGatewayNextHop:    localnetGatewayNextHop,
+		ovn.OvnNodeGatewayIP:         localnetGatewayIP(),
+		ovn.OvnNodeGatewayNextHop:    localnetGatewayNextHop(),
 	}
 	annotations := map[string]map[string]string{
 		ovn.OvnDefaultNetworkGateway: l3GatewayConfig,
 	}
 
-	err = localnetGatewayNAT(ipt, localnetBridgeNextHop, localnetGatewayIP)
+	if config.UseIPv6() {
+		// TODO - IPv6 hack ... for some reason neighbor discovery isn't working here, so hard code a
+		// MAC binding for the gateway IP address for now - need to debug this further
+		_, _, _ = util.RunIP("-6", "neigh", "del", "fd99::2", "dev", "br-nexthop")
+		stdout, stderr, err := util.RunIP("-6", "neigh", "add", "fd99::2", "dev", "br-nexthop", "lladdr", macAddress)
```

Comment:
@russellb can you revisit RE https://github.com/ovn-org/ovn-kubernetes/pull/983 and https://github.com/ovn-org/ovn-kubernetes/commit/75f836d91e1eeadd984089eb3f650734918972cf ? We now use a fixed MAC (see the `localnetGatewayNextHopMac` constant).

Reply:
I think you're suggesting this may not be needed anymore?  That will be easier for me to test once openshift is caught up to include the changes you're referring to.  I could probably get it all applied, but that would have to wait until after the holidays.  If it's not needed anymore, I think it's harmless in the meantime?

Reply:
or would you like me to just remove it, hope it's fine now, and propose it as a new PR later if I determine it's still needed?